app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+
+class UserController extends Controller
+{
+    public function get(User $user): User
+    {
+        return $user;
+    }
+
+    public function store(Request $request)
+    {
+        $request->validate([
+            'id' => 'nullable',
```

Comment:
Is het de bedoeling dat id nullable is?

Reply:
Yep. ID wordt in principe auto-gegenereerd. De enige reden waarom je het handmatig kan invoeren is voor het testen.